test/pages/pages.test.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+/* eslint-disable no-undef */
+/* eslint-disable camelcase */
+const { fetch } = require('@adobe/helix-fetch');
+const assert = require('assert');
+const { JSDOM } = require('jsdom');
+const { dom } = require('@adobe/helix-shared');
+
+const testDomain = process.env.TEST_DOMAIN;
+
+let bases = [];
+let changes = [];
+let base_urls = [];
+
+async function getDoms() {
+  const json = {
+    limit: 10,
+    threshold: 100,
+  };
+  const method = 'post';
+  const res = await fetch('https://adobeioruntime.net/api/v1/web/helix/helix-services/run-query@v2/most-visited', { method, json });
+  if (!res.ok) {
+    assert.fail('test setup failed to gather test urls');
+  }
+  base_urls = (await res.json()).results;
+  // construct array of promises from fetch
+  changes = base_urls.map((obj) => {
+    // eslint-disable-next-line camelcase
+    const { req_url } = obj;
+    const { pathname, hostname } = new URL(req_url);
+    const headers = {
```

Comment:
this is no longer needed with helix-fetch v1.6.2: https://github.com/adobe/helix-fetch/releases/tag/v1.6.2